web/package.json

```diff
@@ -8,7 +8,13 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "test:unit": "jest",
+    "lint": "eslint ./src",
+    "lint:fix": "eslint ./src --fix",
+    "prettier": "prettier src --check",
+    "prettier:fix": "prettier src --write",
+    "format": "yarn prettier && yarn lint",
+    "format:fix": "yarn prettier:fix && yarn lint:fix",
+    "test": "yarn concurrently -g 'yarn:test:*'",
     "start": "VENDOR_WEB_ENV=dev webpack-dev-server --env dev"
```

Comment:
Couldn't figure out how to get prettier to fix with eslint so there are two sets of commands.

`yarn format` runs prettier and eslint 
`yarn format:fix` runs and fixes with prettier and eslint 

The CI will run `yarn format` as a required check. I will be adding unit tests to my work next week and modify the test command to run all of our required tests with concurrently.